src/tutorial/examples.md

```diff
@@ -27,17 +27,25 @@ and <a href="../fem"><i class="fa fa-book"></i> Finite Element Basics</a> pages
 ---
 
 ### <i class="fa fa-check-square-o"></i>&nbsp; High-order methods
-- MFEM includes support for the full *de Rham complex*; $H^1-$conforming (continuous), $H(curl)-$conforming (continuous tangential component), $H(div)-$conforming (continuous normal component) and $L^2-$conforming (discontinuous) energy spaces in 2D and 3D. A compatible high-order de Rham complex on the discrete level can be constructed using the `*_FECollection` with `*` replaced by `H1`, `ND`, `RT` and `L2`, respectively. The first four MFEM examples serve as an introduction on how to construct and use these discrete spaces for the solution of various PDEs.
+- MFEM includes support for the full [*de Rham complex*](https://en.wikipedia.org/wiki/De_Rham_cohomology), $H^1-$conforming (continuous), $H(curl)-$conforming (continuous tangential component), $H(div)-$conforming (continuous normal component), and $L^2-$conforming (discontinuous) energy spaces in 2D and 3D. A compatible high-order de Rham complex on the discrete level can be constructed using the `*_FECollection` with `*` replaced by `H1`, `ND`, `RT`, and `L2`, respectively. The first four MFEM examples serve as an introduction on how to construct and use these discrete spaces for the solution of various PDEs.
 
 - Building the example codes:
     - Make sure that the MFEM source code is already compiled
     - Enter the `examples` directory: `cd ~/mfem/examples` 
-    - Compile the serial version of `example ?` with `make ex?` or the parallel version with `make ex?p`. 
+
+<div class="panel panel-info" style="width:92%; margin-left: auto; margin-right: auto;">
+<div class="panel-heading">
+<h3 class="panel-title"><i class="fa fa-info-circle"></i>&nbsp; Note</h3>
+</div>
+<div class="panel-body">
+Remember to compile each numbered example before executing its sample runs: <code>make ex?</code> for the serial version or <code>make ex?p</code> for the parallel version. You can build multiple examples in the same command: <code>make ex3 ex4 ex3p ex4p</code>
```

Comment:
I added this b/c I had a lot of errors and couldn't run anything after Example 1 until I got to `solvers.md` and realized I had to build the others.